src/java/org/apache/cassandra/dht/Range.java

```diff
@@ -942,6 +946,24 @@ public static <T extends RingPosition<T>> boolean equals(Collection<Range<T>> a,
         return normalize(a).equals(normalize(b));
     }
 
+    public static accord.primitives.Ranges toAccord(String ks, Collection<Range<Token>> ranges)
+    {
+        List<Range<Token>> normalizedRanges = Range.normalize(ranges);
+        TokenRange[] tokenRanges = new TokenRange[normalizedRanges.size()];
+        for (int i = 0; i < normalizedRanges.size(); i++)
+        {
+            Range<Token> tokenRange = normalizedRanges.get(i);
+            AccordRoutingKey left = new TokenKey(ks, tokenRange.left);
+            AccordRoutingKey right = new TokenKey(ks, tokenRange.right);
+            if (tokenRange.left.isMinimum())
+                left = AccordRoutingKey.SentinelKey.min(ks);
+            if (tokenRange.right.isMinimum())
+                right = AccordRoutingKey.SentinelKey.max(ks);
```

Comment:
@bdeggleston mind sanity checking this?  I believe its correct, but just wanted to double check.

Reply:
AccordTopologyUtils already has range conversion code, we should just use that